packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/configure-CloudFront.js

```diff
@@ -61,8 +65,19 @@ async function configure(context) {
       delete context.exeInfo.template.Outputs.CloudFrontDistributionID;
       delete context.exeInfo.template.Outputs.CloudFrontDomainName;
       delete context.exeInfo.template.Outputs.CloudFrontSecureURL;
-      const { AccessControl } = originalTemplate.Resources.S3Bucket.Properties;
+      delete context.exeInfo.template.Outputs.CloudFrontOriginAccessIdentity;
+
+      const {
+        AccessControl,
+        WebsiteConfiguration,
+      } = originalTemplate.Resources.S3Bucket.Properties;
       context.exeInfo.template.Resources.S3Bucket.Properties.AccessControl = AccessControl;
+      context.exeInfo.template.Resources.S3Bucket.Properties.WebsiteConfiguration =
+        WebsiteConfiguration;
+
+      context.print.warning('Static webhosting will be enabled for the hosting bucket once your remove CloudFront Distribution.');
```

Comment:
Quite a few typos in this warning -> "Static web hosting will be enabled for the hosting bucket once you remove CloudFront Distribution."
Does this mean that when Cloudfront is disabled, the contents of the S3 bucket will automatically become public for enabling S3 web hosting?

Reply:
Fixed typos. 
S3 bucket is not public just because you set for static web hosting, it only exposes the contents with static web hosting urls. 
Bucket itself is private, contents are `public-read` when CloudFront is removed.